packaging/fedora/snapd.spec

```diff
@@ -120,7 +120,9 @@ ExclusiveArch:  %{ix86} x86_64 %{arm} aarch64 ppc64le s390x
 
 # If go_compiler is not set to 1, there is no virtual provide. Use golang instead.
 BuildRequires: make
+%if ! 0%{?openEuler}
 BuildRequires:  %{?go_compiler:compiler(go-compiler)}%{!?go_compiler:golang >= 1.9}
```

Comment:
Can you explain what caused the problem here? With this change I'm not quite sure how the dependency on Go would otherwise be expressed in openEuler